components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -1354,17 +1354,21 @@ define([
 
                 var suggestContainer = container.down('div.patient-newlink-container');
                 var suggestInput     = container.down('input[type=text].suggest-patients');
+                suggestInput.placeholder = " type patient name or identifier";
```

Comment:
I see, it's actually looks well without that space, so I'll just remove it
Regarding placeholders, in patient form we have it with both capital and small first letters, I personally prefer small, but it's super minor.